spec/support/shared_examples/shared_examples_for_controllers.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+shared_examples 'ApplicationController#require_login' do
+  it 'receives the #require_login method from ApplicationController' do
+    test_login
+    expect(controller).to receive(:require_login)
+    action
+  end
+end
+
+shared_examples 'ApplicationController#require_logout' do
+  it 'receives the #require_login method from ApplicationController' do
```

Comment:
The message for this `it` is confusing, but i am also not sure where in the app we would require logout specifically.